public/js/showoff.js

```diff
@@ -102,10 +102,14 @@ function setupPreso(load_slides, prefix) {
 */
 }
 
-function loadSlides(load_slides, prefix) {
+function loadSlides(load_slides, prefix, reload) {
```

Comment:
We should... I'd almost prefer that to be part of a larger refactoring effort. What would you prefer?